src/pykx/pandas_api/pandas_meta.py

```diff
@@ -154,6 +154,31 @@ def mean(self, axis: int = 0, numeric_only: bool = False):
             tab
         )
 
+    @api_return
+    def kurt(self, axis: int = 0, numeric_only: bool = False):
+        tab = self
+        if 'Keyed' in str(type(tab)):
+            tab = q.value(tab)
+        if numeric_only:
+            tab = _get_numeric_only_subtable(tab)
+
+        axis_keys = q('{[axis;tab] $[0~axis;cols;`$string til count @] tab}', axis, tab)
+
+        return q(
+            '''{[tab;axis;axis_keys]
+                tab:$[0~axis;(::);flip] value flip tab;
+                kurt:{res: x - avg x;
```

Comment:
Supply `x` and `y` as explicit parameters. Implicit ones are discouraged for large functions.

Reply:
Only needed to supply `x` explicitly. Done